extensions/amp-story/0.1/media-pool.js

```diff
@@ -663,32 +622,42 @@ export class MediaPool {
     domMediaEl.muted = true;
     domMediaEl.setAttribute('muted', '');
     domMediaEl.pause();
+
+    return Promise.resolve();
   }
 
 
   /**
    * Preloads the content of the specified media element in the DOM.
    * @param {!HTMLMediaElement} domMediaEl The media element, found in the DOM,
    *     whose content should be loaded.
+   * @return {!Promise<!HTMLMediaElement>} A promise that is resolved when the
+   *     specified media element has been successfully loaded.
```

Comment:
s/has been successfully loaded/has successfully started preloading?
This comment makes me think all the media content is loaded and ready to play until the end

Reply:
Yep, copy/paste error.  Changed.